packages/plans-grid-next/src/components/shared/action-button/index.tsx

```diff
@@ -188,20 +192,22 @@ const PlanFeatures2023GridActions = ( {
 						{ text }
 					</PlanButton>
 					{ postButtonText && (
-						<span className="plan-features-2023-grid__actions-post-button-text">
-							{ postButtonText }
-						</span>
+						<span className="plans-grid-next__action-button-label">{ postButtonText }</span>
 					) }
 				</>
 			);
 		}
 	}
 
 	return (
-		<div className="plan-features-2023-gridrison__actions">
-			<div className="plan-features-2023-gridrison__actions-buttons">{ actionButton }</div>
+		<div
+			className={ clsx( 'plans-grid-next__action-button', {
+				'plans-grid-next__action-button--fixed-width': isFixedWidth,
```

Comment:
The convention is to have decorators like `.is-bold` or `.is-coloured`, etc. We are not exactly BEM in Calypso.

Also, I don't like how `.is-fixed-width` entails that it's `200px`. Where is this being used? Perhaps we shouldn't be looking at a decorator but a wrapper (parent reference) instead.

Reply:
> The convention is to have decorators like .is-bold or .is-coloured, etc. 

Updated: 78482b6

Reply:
> Also, I don't like how .is-fixed-width entails that it's 200px. Where is this being used? 

The fixed width is applied in the 'spotlight' panel.

> Perhaps we shouldn't be looking at a decorator but a wrapper (parent reference) instead.

Updated to qualify the selector based on the 'spotlight' parent: 8db6778. Is that what you had in mind?

I'd personally avoid this approach -- components shouldn't have contextual knowledge about parent component(s). It does seems to be an accepted styling pattern in `plans-grid-next` though 🤷.

Ideally we would utilise a `css-module`-like library to maintain styling boundaries.

Reply:
> I'd personally avoid this approach -- components shouldn't have contextual knowledge about parent component(s). It does seems to be an accepted styling pattern in plans-grid-next though 🤷

I actually came here now to comment on the same thing. 

Instead of having these parent references, it might be better to let the parent override/tailor the defaults on its end. In principle, this makes sense, and will get things moving towards more self-contained/isolated components that are consumed by broader components like the "features grid" or "comparison grid".

Reply:
Followup comment: https://github.com/Automattic/wp-calypso/pull/93312#discussion_r1710654191